naabu: add page (#12206)
tldr-pages/tldr: pages/common/naabu.md

# naabu

> > A fast port scanner written in go with a focus on reliability and simplicity.
> > Some features only activate when naabu is run with root privileges such as SYN scan.

Reviewer: ```suggestion
> A fast port scanner written in Go with a focus on reliability and simplicity.
> Note: Some features are only activated when `naabu` is run with root privileges such as SYN scan.
```
I have a few suggestions here:

1. We can capitalize Go.
2. We can convert the second line to a note and enclose the command name inside backticks.

> More information: <https://github.com/projectdiscovery/naabu>.

- Run a SYN scan against default (top 100) ports of remote host:

`sudo naabu -host {{host}}`

> - Display available network interfaces and public ip of local host:

Reviewer: ```suggestion
- Display available network interfaces and public IP address of the local host:
```

`naabu -interface-list`

> - Scan all ports of remote host (CONNECT scan without sudo):

Reviewer: ```suggestion
- Scan all ports of the remote host (CONNECT scan without `sudo`):
```

`naabu -p - -host {{host}}`

> - Scan top 1000 ports of remote host:

Reviewer: ```suggestion
- Scan the top 1000 ports of the remote host:
```

`naabu -top-ports 1000 -host {{host}}`

> - Scan tcp ports 80,443 and udp port 53 of remote host:

Reviewer: ```suggestion
- Scan TCP ports 80,443 and UDP port 53 of the remote host:
```
We generally capitalize protocols and abbreviations.

`naabu -p 80,443,u:53 -host {{host}}`

> - Show CDN type remote host is using, if any:

Reviewer: ```suggestion
- Show CDN type the remote host is using, if any:
```
